variables.tf

```diff
@@ -33,78 +37,91 @@ variable "route53_role_arn" {
 variable "account_name_regex" {
   default     = "^$"
   description = "A regular expression that will be applied against the names of all non-master accounts in the AWS organization.  If the name of an account matches the regular expression, that account will be queried for egress IP addresses to publish.  The default value should not match any valid account name."
+  nullable    = false
   type        = string
 }
 
 variable "application_tag" {
   default     = "Application"
   description = "The name of the AWS tag whose value represents the application associated with an IP address."
+  nullable    = false
   type        = string
 }
 
 variable "aws_region" {
   default     = "us-east-1"
   description = "The AWS region to deploy into (e.g. us-east-1)."
+  nullable    = false
   type        = string
 }
 
 variable "cloudfront_distribution_oac_description" {
   default     = "Allow CloudFront to securely read from an S3 bucket."
   description = "The description to apply to the CloudFront Origin Access Control."
+  nullable    = false
   type        = string
 }
 
 variable "cloudfront_distribution_oac_name" {
   default     = "publish-egress-ip-s3-distribution"
   description = "The name for the CloudFront Origin Access Control."
+  nullable    = false
   type        = string
 }
 
 variable "domain_aliases" {
   default     = []
   description = "A list of domain aliases to associate with the CloudFront distribution.  These domains must be included (as subject alternative names) in the certificate used by the distribution."
+  nullable    = false
   type        = list(string)
 }
 
 variable "ec2_read_role_name" {
   default     = "EC2ReadOnly"
   description = "The name of the IAM role that allows read access to the necessary EC2 attributes.  Note that this role must exist in each account that you want to query."
+  nullable    = false
   type        = string
 }
 
 variable "extraorg_account_ids" {
   default     = []
   description = "A list of AWS account IDs corresponding to \"extra\" accounts that you want to query for egress IPs to publish."
+  nullable    = false
   type        = list(string)
 }
 
 variable "file_configs" {
   default     = []
   type        = list(object({ app_regex = string, description = string, filename = string, static_ips = list(string) }))
+  nullable    = false
```

Comment:
Can you please fix the alphabetical ordering of the keys here?

Reply:
Please see commit 9f8feec.